### ISSUE TITLE
Unable to change mining speed using UseTimeMultiplier

### ISSUE THREAD

**Issue by ghost** [bluemagic123/tModLoader#2509]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Mod capability as a Modder

### Description

I'm trying to change mining speed using UseTimeMultiplier, but it didn't work
i've tried UseSpeedMultiplier, and it didn't work too

i've seen a mod that used this to change mining speed, but it is for 1.3 only
and i didn't found other function that would change mining speed

### Log File

[client.log](https://github.com/tModLoader/tModLoader/files/8849091/client.log)

### Steps to reproduce

1.
```
public override float UseTimeMultiplier(Item item) {
            return 5f;
        }
```

2. use any pickaxe/hammer/axe

### Expected Behavior

faster mining speed

### Actual Behavior

nothing

**Comment by Chicken-Bones** [comment]
#1390 needs updating to address this

**Comment by andrewlord607** [comment]
And most likely [this](https://github.com/tModLoader/tModLoader/pull/1392) too?

**Comment by Chicken-Bones** [comment]
Fixed by ad03f8ce06ea77a02753d380b6e6a9598843ce25
